Source/WebKit/UIProcess/Cocoa/WebProcessPoolCocoa.mm

```diff
@@ -249,6 +250,20 @@ static AccessibilityPreferences accessibilityPreferences()
 }
 #endif
 
+static void logProcessPoolState(const WebProcessPool& pool)
+{
+    for (const auto& process : pool.processes()) {
+        WTF::TextStream stream;
+        stream << process;
+
+        String domain = "unknown"_s;
+        if (process->optionalRegistrableDomain())
+            domain = process->optionalRegistrableDomain()->string();
```

Comment:
maybe we could do:
```
else
    domain = "unknown"_s;
```
To avoid always constructing a string from "unknown"_s even when there is a registrable domain.